.github/workflows/webrtc-load-test.yaml

```diff
@@ -26,10 +26,16 @@ on:
         required: true
         default: '{"us-central1":20,"europe-west2":20,"asia-southeast1":20}'
       streamer-input-file:
-        description: File to stream on the load test
+        description: File to ingest
         type: string
         required: true
         default: 'https://storage.googleapis.com/lp_testharness_assets/countdown_720p_30fps_2sGOP_noBframes_5min.mp4'
+      playback-protocol:
+        description: Playback protocol
+        required: true
+        type: choice
+        options: [ "WebRTC", "HLS" ]
```

Comment:
@victorges You should make `name:`, `run-name:` and discord messages being parametrized by `${{ inputs.playback-protocol }}` and rename `webrtc-load-test.yaml` into `load-test.yaml` as it's now more generalized.

Reply:
https://github.com/livepeer/stream-tester/pull/346